client/overview/setup-real-payments.tsx

```diff
@@ -63,15 +64,20 @@ const SetupRealPayments: React.FC = () => {
 					</div>
 				</CardBody>
 				<CardFooter className="wcpay-setup-real-payments__footer">
-					<Button isPrimary onClick={ () => setModalVisible( true ) }>
-						{ __( 'Set up payments', 'woocommerce-payments' ) }
-					</Button>
+					<Flex align="center" justify="flex-start">
+						<Button
+							isSecondary={ true }
```

Comment:
I see that this attribute is marked as deprecated 
<img width="713" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/79862886/b708f582-76ac-4b47-b080-b3715c2f3843">
We could use `variant={ 'secondary' }`, I see it's used in our codebase, e.g. [here](https://github.com/Automattic/woocommerce-payments/blob/d13d3268f26ce9238fd24f315e30c8332b4e800c/client/disputes/index.tsx#L301-L301)

Reply:
Ah, thank you! I noticed that too but I wasn't sure what the alternative was. Will make that adjustment before merge! 🙂

Reply:
Addressed in 8963da319.